instagram/src/components/PostsContainer/Post.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Posts.css';
+import CommentSection from '../CommentSection/CommentSectionContainer';
+
+class Post extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      post: props.post,
+      liked: false
+    }
+  }
+
+  incrementLike = (event) => {
+    event.preventDefault();
+
+    const post = JSON.parse(JSON.stringify(this.state.post))
+    if (this.state.liked) {
+      post.likes -= 1;
+      this.setState({ post, liked: false });
+    } else {
+      post.likes += 1;
+      this.setState({ post, liked: true });
```

Comment:
Nice job adding the liked insta button here as part of the stretch goal.❤️ I like how you included the logic to decrement the value of likes once you click on the heart again. 💯